src/components/Autocompete.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import '../App.scss';
+import { Person } from '../types/Person';
+
+type Props = {
+  dropDownList: Person[]
+  onSelect: (slug: string) => void
+};
+
+export const Autocomplete: React.FC<Props> = ({ dropDownList, onSelect }) => {
```

Comment:
![image](https://user-images.githubusercontent.com/65460971/214267223-32f8c0b7-4a48-4e90-9d4a-8731b1a19326.png)

I ran some tests,
Whenever anything changes in input field autocomplete component is being rendered. 
You can use Memo for that component